Cannot add reduce options when using MultipleRequestBuilder

- CloudantClient (java-cloudant) version(s) that are affected by this issue.
    * 2.x
- Java version (including vendor and platform).
    * Any
- If you're using the optional okhttp dependency.
    * N/A
- A small code sample that demonstrates the issue.

 ```java
ViewRequestBuilder viewRequestBuilder = db.getViewRequestBuilder("design", "view"); 
ViewMultipleRequest<String, String> viewRequest = viewRequestBuilder
        .newMultipleRequest(Key.Type.STRING, String.class)
        .includeDocs(true)
        .reduce(false)
        .build();   
```

will cause a compilation error 
>error: cannot find symbol            
    .reduce(false)

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,3 +1,7 @@
+# Unreleased
+- [NEW] Enabled `reduce` and other reduce related parameters to be set when using
+  `MultipleRequestBuilder`.
+
 # 2.5.1 (2016-07-19)
 - [IMPROVED] Made the 429 response code backoff optional and configurable. To enable the backoff add
   an instance of a `Replay429Interceptor` with the desired number of retries and initial backoff:

--- a/cloudant-client/src/main/java/com/cloudant/client/api/views/MultipleRequestBuilder.java
+++ b/cloudant-client/src/main/java/com/cloudant/client/api/views/MultipleRequestBuilder.java
@@ -47,7 +47,8 @@ package com.cloudant.client.api.views;
  * @since 2.0.0
  */
 public interface MultipleRequestBuilder<K, V> extends RequestBuilder<MultipleRequestBuilder<K, V>>,
-        SettableViewParameters.Unpaginated<K, MultipleRequestBuilder<K, V>> {
+        SettableViewParameters.Unpaginated<K, MultipleRequestBuilder<K, V>>,
+        SettableViewParameters.Reduceable<K, MultipleRequestBuilder<K, V>> {
 
     /**
      * Adds a query request to this MultipleRequestBuilder.

--- a/cloudant-client/src/test/java/com/cloudant/tests/ViewsTest.java
+++ b/cloudant-client/src/test/java/com/cloudant/tests/ViewsTest.java
@@ -814,6 +814,35 @@ public class ViewsTest {
     }
 
     /**
+     * Validate that it is possible to POST multiple requests to a view query mixing reduced and not
+     * reduced queries and that the results of each request are as expected.
+     *
+     * @throws IOException
+     */
+    @Test
+    @Category(RequiresCloudant.class)
+    public void multiRequestMixedReduced() throws IOException {
+        init();
+        ViewMultipleRequest<String, Object> multi = db.getViewRequestBuilder("example", "by_tag")
+                .newMultipleRequest(Key.Type.STRING, Object.class)
+                /* add includeDocs args after https://issues.apache.org/jira/browse/COUCHDB-3070 */
+                .reduce(false).keys("java")/*.includeDocs(true)*/.add()
+                .reduce(true)/*.includeDocs(false)*/.add()
+                .build();
+
+        List<ViewResponse<String, Object>> responses = multi.getViewResponses();
+        assertEquals("There should be 2 respones for 2 requests", 2, responses.size());
+
+        List<String> javaTagKeys = responses.get(0).getKeys();
+        assertEquals("There should be 1 java tag result", 1, javaTagKeys.size());
+        assertEquals("The key should be java", "java", javaTagKeys.get(0));
+
+        List<Object> allTagsReduced = responses.get(1).getValues();
+        assertEquals("There should be 1 reduced result", 1, allTagsReduced.size());
+        assertEquals("The result should be 4", 4, ((Number) allTagsReduced.get(0)).intValue());
+    }
+
+    /**
      * Assert that no additional pages are available on an unpaginated request even if additional
      * results are available.
      *

